src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+}
+  
+ul {
+  margin: 0;
+  padding: 0;
+}
+  
+.stars {
+  display: flex;
+  list-style: none;
+}
+  
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+  
+.stars--1 > .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--2 > .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--2 > .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--3 > .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--3 > .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--3 > .stars__star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--4 > .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--4 > .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--4 > .stars__star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--4 > .stars__star:nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--5 > .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--5 > .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--5 > .stars__star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--5 > .stars__star:nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+  
+.stars--5 > .stars__star:nth-child(5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Let's combine rules `,` and try to use `nth-child` with `n`. For example to select 2 first stars `nth-child(-n + 2)`